TopicsImplementations/src/topiczerotwo/EntryPointClass.java

```diff
@@ -0,0 +1,36 @@
+package topiczerotwo;
+
+import java.io.IOException;
+
+/**
+ * Main class to test Abstract factory pattern with DB connections.
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public class EntryPointClass {
+
+	// Types of DB
+	private static final String ORACLE = "ORACLE";
```

Comment:
Maybe you can create an enum for this instead of depending on string matching. Using this if a dev does a typo mistake sometimes is difficult to find the error. 
Create one enum for DB types (ORACLE, SQL) and another for the different SQL instances (MYSQL, POSTGRESQL, SQLITE)